src/testing/iotest/reader.go

```diff
@@ -68,6 +68,7 @@ func (r *dataErrReader) Read(p []byte) (n int, err error) {
 	return
 }
 
+// ErrTimeout is a fake timeout error
```

Comment:
doc comments should end with `.`s:
```suggestion
// ErrTimeout is a fake timeout error.
```